testnet/docker-compose.dev-testnet.yml

```diff
@@ -41,9 +41,9 @@ services:
     ]
 
   enclave:
-    privileged: true
-    volumes:
-      - /dev/sgx:/dev/sgx
+    devices:
+      - /dev/sgx_enclave
+      - /dev/sgx_provision
```

Comment:
Is this for the enclave as well as edgelessdb then?

Reply:
Trying it out actually